.travis.yml

```diff
@@ -0,0 +1 @@
+language: node_js
```

Comment:
This means 'run tests in node js environment, do `npm install` to install dependencies and `npm test` to test project'.